.github/workflows/build-and-deploy.yml

```diff
@@ -205,6 +205,16 @@ jobs:
         with:
           password: ${{ secrets.PYPI_TOKEN }}
 
+  deploy-rust:
+    runs-on: kuzu-self-hosted-linux-building
+    steps:
+      - uses: actions/checkout@v3
+
+      - run: cargo publish
```

Comment:
For cargo publish, is there a way to dry-run or deploy to a test environment similar to npm or pypi? If so, we should do dry run unless `isDeploy` is set to true.

Reply:
Yes, there's a `--dry-run` option which skips uploading. I don't think there's a test registry like test.pypi.org, but we could just upload the packaged crate file to github.

Reply:
OK. Please add the dry-run based on the flag. We should always upload the packaged files to GitHub as well for debugging purpose in case something goes wrong and the deployment fails.